GameEngine/EngineInternal.cs

```diff
@@ -61,18 +65,22 @@ public void StartEngine()
                     }
                 }
 
-                if (playerCharacter.IsDead())
+                foreach (var playerCharacter in playerCharacters)
                 {
-                    playerCharacter.SendMessage();
-                    playerCharacter.SendMessage("You have died. Please press a key.");
-                    Console.ReadKey();
-                    RunGameLoop = false;
+                    if (playerCharacter.IsDead())
```

Comment:
So the rule is that if any player dies, the game is over?

Reply:
yes, the main idea of where i'm going here is to make a game where all players need to be alive.